### PR TITLE
Add test comments in core

### PR DESCRIPTION
**Pull request by askeko**

### CODE REVIEW

**Review by Bliztle — APPROVED** [review]

**Review by Bliztle — APPROVED** [review]
LGTM (Let's Gamble, Try Merging)

File: pmp-core/src/test/java/dk/nykredit/pmp/core/audit_log/TestAuditLogSerializer.java [hunk 1]
```diff
@@ -17,24 +16,39 @@
...
         entity.setChangeType(ChangeType.PARAMETER_CHANGE);
         entity.setParameterName("test1");
         entity.setParameterType("String");
         entity.setOldValue("first");
         entity.setNewValue("second");
 
+        /**
+         * Serialize change entity to json and check that the correct values are
+         * present.
+         * Throw exception if serialization fails.
```
Comment (TheMagzuz) [inline pmp-core/src/test/java/dk/nykredit/pmp/core/audit_log/TestAuditLogSerializer.java:50]: This is also try-with-resources so it automatically closes the `parser` object

File: pmp-core/src/test/java/dk/nykredit/pmp/core/audit_log/TestAuditLogSerializer.java [hunk 2]
```diff
@@ -47,56 +61,76 @@ public void testSerializeParameterChangeEntity() {
...
         String json;
         try {
             json = mapper.writeValueAsString(entry);
         } catch (JsonProcessingException e) {
             throw new RuntimeException(e);
         }
 
+        /**
+         * Deserialize json and check that the correct values are present.
+         * Throw exception if deserialization fails (should not happen).
```
Comment (TheMagzuz) [inline pmp-core/src/test/java/dk/nykredit/pmp/core/audit_log/TestAuditLogSerializer.java:102]: Also a try-with-resources

File: pmp-core/src/test/java/dk/nykredit/pmp/core/commit/TestCommitRevert.java [hunk 3]
```diff
@@ -147,24 +190,42 @@ void testDontRevertParametersWithNewerChanges() {
...
+        // Add changes to revert commit.
         revertCommit.setChanges(changes2);
+
+        // Apply revert commit.
         commitDirector.apply(revertCommit);
 
+        // Stop transaction.
         parameterService.getRepository().endTransaction();
 
+        // Check that the parameter change is not reve>rted as the parameter changed in
```
Comment (TheMagzuz) [inline pmp-core/src/test/java/dk/nykredit/pmp/core/commit/TestCommitRevert.java:206]: stray `>`

**Review by TheMagzuz — CHANGES_REQUESTED** [review]

**Review by TheMagzuz — APPROVED** [review]

**Pull request merged**